packages/block-library/src/list/style.scss

```diff
@@ -0,0 +1,17 @@
+ol {
+	&.ol-type-is-1 {
```

Comment:
Is this needed on the front-end as well? I thought it was only needed in the editor because of an admin CSS rule.

Reply:
Yes, it is needed. Otherwise, the list in the preview will not be same as the list in the editor. There is also an admin CSS rule in the preview page, which will override the list attribute `<ol type="1|a|A|i|I">`
![image](https://user-images.githubusercontent.com/24849874/56830918-b2925280-6835-11e9-96c4-174f7b80a20c.png)

Reply:
So the preview page is different from the final page? Strange that it has admin CSS. Do you know why?

Reply:
Both the final page and preview page have such an admin rule, which will override the `type` attributes. I have no idea why it has the rule.

Reply:
I don't see any admin rules. But I do see theme rules that set `ol` to `decimal`. That's unfortunate. Can we not force set `ol[type="A"]` etc. to override the theme style, instead of using a class? That seems to work?

Reply:
Ah, I remember you comment about it being case insensitive. :)

Reply:
To be honest, in that case, I think we should just let it be overridden by the theme and encourage themes to not force decimal list styling on `ol` elements. I don't know why this is done. As soon as this PR is merged, the core themes should be updated. Does this sound reasonable @jasmussen @kjellr?